draft-ietf-v6ops-464xlaton.xml

```diff
@@ -302,20 +306,32 @@ That improves the chances of the protocol working via CLAT even if CLAT is not a
 </li>
 </ul>
 
+<t>
+The node SHOULD treat SLAAC-generated CLAT addresses as temporary addresses (<xref target="RFC8981"/>). In particular:
+</t>
+<ul>
+<li>
+<t>
+The node SHOULD generate new CLAT addresses over time, as described in Sections 3.4 and 3.6 of <xref target="RFC8981"/>.
+</t>
+</li>
+<li>
 <t>
 The node SHOULD generate a different interface id for the CLAT address when connecting to different networks, even if the NAT64 prefix and the local IPv4 CLAT address do not change. 
-To achieve that, the node SHOULD generate a random checksum-neutral CLAT address every time.
+To achieve that, the node SHOULD generate a random checksum-neutral CLAT address every time the network attachement changes to another network.
 </t>
+</li>
+</ul>
 
 <section anchor="mhoming">
 <name>CLAT and Multiple Prefixes per Interface</name>
 <t>
-The CLAT instance SHOULD obtain a dedicated IPv6 address (or addresses, if a dedicated prefix model is used) from each prefix used to configure IPv6 addresses on the given interface. When selecting an address to be used as a source address for the translated packet, the following rules are applied:
+The CLAT instance SHOULD obtain a dedicated IPv6 address (or addresses, if a dedicated prefix model is used) from each global prefix used to configure IPv6 addresses on the given interface. When selecting an address to be used as a source address for the translated packet, the following rules are applied:
 </t>
 <ul>
 <li>
 <t>
-All packets with the same IPv4 5-tuple MUST be translated to the same IPv6 address.
+All packets with the same IPv4 5-tuple MUST be translated to the same valid (<xref target="RFC4862"/>) IPv6 address.
```

Comment:
Why specify this? No objection, just curious what led to this clarification.

Reply:
Rephrased, PTAL

Reply:
LGTM, thanks!